kotlin-java8/pom.xml

```diff
@@ -13,6 +13,7 @@
 
     <properties>
         <kotlin.version>1.2.41</kotlin.version>
+        <project.Automatic-Module-Name>io.cucumber.kotlin.java8</project.Automatic-Module-Name>
```

Comment:
This module is excluded from release. It doesn't need the module name.

Reply:
removed